src/localization/translations/pam.json

```
@@ -0,0 +1,28 @@
+{
+  "hero": {
+    "title": "Malaus a Pagdatang king BetterGov.ph",
+    "subtitle": "Ing volunteer-run portal ning Republika ning Pilipinas. Manintun impormasiun, pag-access kareng serbisyu ning kapamahalan, manatili updated kareng pekabayung balita tungkul king Pilipinas."
+  },
+  "services": {
+    "title": "Deng Sikat a Serbisyu"
+  },
+  "weather": {
+    "title": "Deng Update king Panaun"
+  },
+  "forex": {
+    "title": "Pamagpalit Pera"
+  },
+  "news": {
+    "title": "Pekabayung Balita ampong Updates"
+  },
+  "footer": {
+    "copyright": "© 2025 BetterGov. Ngan laman public domain la pin detang mebangit a aliwa."
```

Somewhat rough translation in the end. Its hard to translate this part.
```suggestion
    "copyright": "© 2025 BetterGov. Ing eganaganang laman at impormasyun ay parti ning publiku maliban mu nung ali ya makasulat."
```